checks/ci_tests.go

```diff
@@ -35,13 +46,6 @@ func CITests(c *checker.CheckRequest) checker.CheckResult {
 		return checker.MakeRetryResult(ciTestsStr, err)
 	}
 
-	const (
-		// States for which CI system is in use.
-		unknown = iota
-		githubStatuses
-		githubCheckRuns
-	)
-
 	usedSystem := unknown
```

Comment:
Optional: if you are planning to update the consts above, update this to:

`usedSystem state = unknown`

And further below, use `switch` instead of `if`. i.e

```
switch(usedSystem) {
  case unknown:
    ...
  case githubStatuses:
    ...
  case githubCheckRuns:
    ...
}
```

Reply:
I wasn't able to use a `switch` here because if `usedSystem` is unknown, I need to execute both `if` blocks. This did give me the idea to change the `if` to use `!=` which I think makes it more readable.

Reply:
Thanks!